quickget

```diff
@@ -362,6 +362,56 @@ list_isos() {
     exit 0
 }
 
+test_isos() {
+    local test_iso_url="1"
+    local DIR="/dev/null"
+    local FUNC
+    local OPTION
+    local OS
+
+    check_it() {
+        validate_release releases_"${OS}"
+        URL=$(get_"${OS}" "${OPTION}" | cut_1)
+        GOOD=$(wget -q --spider "${URL}" && echo 'OK' || echo 'ERROR!')
+        if [[ "${GOOD}" == "OK" ]]; then
+            echo "OK - ${OS},${RELEASE},${OPTION},${URL}"
+        else
+            echo "ERROR - ${OS},${RELEASE},${OPTION},${URL}"
+        fi
+    }
+
+    for OS in $(os_support); do
+        case ${OS} in
+          *ubuntu-server*) FUNC="ubuntu-server";;
+          *ubuntu*) FUNC="ubuntu";;
+          *) FUNC="${OS}";;
+        esac
+        for RELEASE in $("releases_${FUNC}" | sed -Ee 's/eol-\S+//g' ); do # hide eol releases
+            # If the OS has an editions_() function, use it.
+            if [[ $(type -t "editions_${OS}") == function ]]; then
+                for OPTION in $(editions_"${OS}"); do
+                    check_it
+                done
+            elif [[ "${OS}" == "windows"* ]]; then
```

Comment:
You can't check every language for Windows, Microsoft will block you from submitting requests for more links after 4-5 attempts per release within a ~30 min period. Instead, you should just check 1 or 2 languages per release. That excludes windows server releases & windows 10-ltsc, though, since there's no API that has to be used.